src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Colors[] colors = Colors.values();
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Colors.values().length);
+        return colors[index].toString();
```

Comment:
Please, pay attention to that [rule](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#you-should-create-several-random-figures-so-you-will-use-a-loop-please-dont-create-a-new-figuresupplier-inside-the-loop).

Reply:
Please, don't use toString when working with enums, here the detailed description of the issue: https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants